lib/src/test/java/co/nimblehq/alpine_lib/ExampleUnitTest.kt

```diff
@@ -0,0 +1,17 @@
+package co.nimblehq.alpine_lib
+
+import org.junit.Test
+
+import org.junit.Assert.*
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+class ExampleUnitTest {
+    @Test
+    fun addition_isCorrect() {
+        assertEquals(4, 2 + 2)
+    }
+}
```

Comment:
Same for this file https://github.com/nimblehq/alpine/pull/8#discussion_r923091747